modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,62 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infastructure.
+
+[IMPORTANT]
+====
+You need to ensure that your BMC supports all of the redfish APIs prior to installation.
+====
+
+**List of redfish APIs:**
+
+.PowerOn
```

Comment:
I would consider using a [description list](https://asciidoctor.org/docs/asciidoc-writers-guide/#description-lists) here (or something else that doesn't make the API objects into red subheader text - looks a bit odd imo). If you go that route, you could make line 77 (List of redfish APIs) a subheading instead :nerd_face: (`.List of redfish APIs`)

Reply:
Yeah testing out the formating for this one was a lot of trial and error 😆. I did not know about the descriptions list, I'll try that and a few things, thank you!